doc/generic/pgf/pgfmanual-en-tikz-arrows.tex

```diff
@@ -796,7 +796,7 @@ \subsubsection{Coloring}
     filled.
 \end{key}
 
-\begin{key}{/pgf/arrow keys/fill=\meta{color or |none|}}
+\begin{key}{/pgf/arrow keys/fill=\meta{color or |none| or empty}}
```

Comment:
Since now it supports no value, maybe it can be documented similar to `/tikz/fill`.
https://github.com/pgf-tikz/pgf/blob/44f8137449b34f62bc6371bd442ae5cc98f60f18/doc/generic/pgf/pgfmanual-en-tikz-actions.tex#L649

```suggestion
\begin{key}{/pgf/arrow keys/fill=\meta{color} (default is \normalfont last fill color}
```
![image](https://github.com/user-attachments/assets/01774eec-57ec-4642-977e-2a44cf8a8062)

PS: I didn't check whether the wording "last fill color" (taken from new paragraph in doc) is accurate or not.

Reply:
Using `fill=.`, I'm afraid the dot doesn't mean the current fill color, but the stroke color of current path, see example in https://github.com/pgf-tikz/pgf/issues/1348#issuecomment-2222185827.

In other words, currently `/pgf/arrow keys/color` and unknown arrow keys interpreted as color names/expressions never update `.`. See [`\pgf@arrows@color@setup`](https://github.com/pgf-tikz/pgf/blob/44f8137449b34f62bc6371bd442ae5cc98f60f18/tex/generic/pgf/basiclayer/pgfcorearrows.code.tex#L923-L935).